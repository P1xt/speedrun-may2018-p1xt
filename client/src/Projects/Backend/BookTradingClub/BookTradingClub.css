.BookTradingClub {
  text-align: left;
  margin-top: 40px; }
  .BookTradingClub > h1 {
    text-transform: none; }
