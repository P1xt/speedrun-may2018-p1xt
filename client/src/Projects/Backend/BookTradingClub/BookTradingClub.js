import React from "react";
import { Container, Row, Col } from "reactstrap";

import "./BookTradingClub.css";

const BookTradingClub = () => {
  return (
    <div className="BookTradingClub">
      <Container>
        <Row>
          <Col>
            <h1>Book Trading Club</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default BookTradingClub;
