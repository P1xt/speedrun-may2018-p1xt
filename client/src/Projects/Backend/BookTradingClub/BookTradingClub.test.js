import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import BookTradingClub from "./BookTradingClub";

describe("<BookTradingClub />", () => {
  it("renders without crashing", () => {
    shallow(<BookTradingClub />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<BookTradingClub />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
