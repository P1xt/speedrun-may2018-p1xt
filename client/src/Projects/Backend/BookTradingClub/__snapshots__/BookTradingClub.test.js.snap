// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BookTradingClub /> renders correctly 1`] = `
<div
  className="BookTradingClub"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Book Trading Club
        </h1>
      </div>
    </div>
  </div>
</div>
`;
